aria/cli/commands/services.py

```diff
@@ -177,3 +179,31 @@ def inputs(service_name, model_storage, logger):
         logger.info(inputs_string.getvalue())
     else:
         logger.info('\tNo inputs')
+
+
+@services.command(name='display',
+                  short_help='Display service information')
+@aria.argument('service-name')
+@aria.options.verbose()
+@aria.options.display_json
+@aria.options.display_yaml
+@aria.options.display_graph
+@aria.pass_model_storage
+@aria.pass_logger
+def display(service_name, model_storage, json, yaml, graph, logger):
```

Comment:
nitpicking that model_storage should probably sit after `graph`, before `logger`